packages/web-react/src/hooks/useStyleUtilities.ts

```diff
@@ -0,0 +1,67 @@
+import { SpacingStyleProp } from '../constants';
+import {
+  BREAKPOINT_MOBILE,
+  BreakpointToken,
+  STYLE_SPACING_AUTO,
+  SpaceToken,
+  StyleProps,
+  StyleSpacingAuto,
+} from '../types';
+import { applyClassNamePrefix } from '../utils';
+
+export type StyleUtilitiesResult = {
+  styleUtilities: string[];
+  props: StyleProps;
+};
+
+const normalizeSpacingValue = (value: string): string =>
+  value === STYLE_SPACING_AUTO ? STYLE_SPACING_AUTO : value.replace(/[^0-9]/g, '');
+
+const processBreakpointProperties = (
+  utilityName: string,
+  propValue: Partial<Record<BreakpointToken, SpaceToken | StyleSpacingAuto>>,
+  prefix: string | null | undefined,
+  accumulatedUtilities: string[] = [],
+) => {
+  return Object.keys(propValue).reduce((accumulatedBreakpointUtilities: string[], breakpoint: string) => {
+    const breakpointValue = propValue[breakpoint as keyof typeof propValue];
```

Comment:
```suggestion
    const breakpointValue: string = propValue[breakpoint as keyof typeof propValue];
```
Try this a maybe you can remove `as string` on the line below.

Reply:
nope, not working